element_calcium_imaging/imaging.py

```diff
@@ -152,8 +152,23 @@ def make(self, key):
             else:
                 raise NotImplementedError('Unknown method: {}'.format(method))
         elif task_mode == 'trigger':
-            raise NotImplementedError(f'Automatic triggering of {method} analysis'
-                                      f' is not yet supported')
+            method = (ProcessingTask * ProcessingParamSet * ProcessingMethod * scan.Scan & key).fetch1('processing_method') 
+            if method == 'suite2p':
+                import suite2p
+
+                suite2p_params = (ProcessingTask * ProcessingParamSet & key).fetch1('params')
+                suite2p_params['save_path0'] = (ProcessingTask & key).fetch1('processing_output_dir')
+                suite2p_params['save_path0'] = (get_imaging_root_data_dir() / Path(suite2p_params['save_path0'])).as_posix()
```

Comment:
should also use `find_full_path` here

Reply:
I tried using the `find_full_path` but it didn't work.